LocalSettings.php

```diff
@@ -4710,7 +4710,7 @@
 		'+bigforestwiki' => [
 			'editvoter',
 		],
-	        '+brandonwmwiki' => [
+			'+brandonwmwiki' => [
```

Comment:
Probably should be fixed to a single indent

Reply:
Thanks; fixed